website/moneybirdsynchronization/management/commands/statementsync.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand
+import datetime
+from moneybirdsynchronization.services import MoneybirdAPIService
+from payments.models import Payment
+from moneybirdsynchronization.administration import HttpsAdministration
+from thaliawebsite import settings
+
+class Command(BaseCommand):
+    """This command needs to be executed every day to sync the bookkeeping with MoneyBird."""
+
+    def handle(self, *args, **options):
+
+        payment_identifiers = {
+            Payment.TPAY: "ThaliaPay",
+            Payment.CASH: "cashtanje",
```

Comment:
Let's not hardcode the values here, but put this mapping in settings.py

Reply:
How exactly? apps are not loaded when settings.py is initialised so making this exact mapping there won't work

Reply:
hmmm that's right... in that case, I would not take `Payment.TPAY` but the literal values these constants represent: `"tpay"`, `"cash"`, `"card"`